Layout-with-Position/css-positioning_layout-with-position/css/styles.css

```diff
@@ -68,14 +86,20 @@ body {
   padding: 6px 12px;
   border-radius: 50%;
   box-shadow: rgba(0, 0, 0, 0.24) 0 3px 8px;
+  position: absolute;
+  top : 30px;
+
+  position: sticky;
+  bottom: 70px;
+  transform: translatex(700%);
```

Comment:
Not sure, what this looks like, haha. If it works, then it's technically not wrong. ;) But surely an unusual way.